aviary/utils/engine_deck_conversion.py

```diff
@@ -501,6 +501,13 @@ def _make_structured_grid(data, method="lagrange3", fields=["thrust", "fuelflow"
 
             # would explicitly use lagrange3 here to mimic GASP, but some engine
             # decks may not have enough points per dimension
+            # For GASP engine deck, try to provide at least 4 Mach numbers.
+            # For GASP_TP engine deck, try to provide at least 4 advanced ratios
```

Comment:
```suggestion
            # For GASP_TP engine deck, try to provide at least 4 Mach numbers
```

advance ratio isn't a variable in the GASP turboshaft decks, we're still checking Mach for number of points here

Reply:
What I meant was the `GASP_TP` engine deck before it was converted to Aviary engine deck. In GASP, it is `advanced ratio`, not `Mach number`.